CMakeLists.txt

```diff
@@ -18,12 +18,28 @@ else ()
 endif ()
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
+
 # options
 option(BUILD_SHARED_LIBS "build shared lib" ON)
 option(MMDEPLOY_BUILD_MODEL_OPTIMIZER "build MMDeploy's model optimizer" ON)
 option(MMDEPLOY_BUILD_SDK "build MMDeploy's SDK" OFF)
 option(MMDEPLOY_ZIP_MODEL "support sdk model in zip format" OFF)
 option(MMDEPLOY_BUILD_TEST "build MMDeploy's csrc's unittest" OFF)
+option(ANDROID_FP16_ACCELERATE "fp16 accelerate for android platform" ON)
```

Comment:
Are the three options necessary?
For evaluation, it can be done outside SDK. 
For FP16, isn't it only related to the model?
For vulkan, I think it's better to let `Device` worry about it.

Reply:
> Are the three options necessary? For evaluation, it can be done outside SDK. For FP16, isn't it only related to the model? For vulkan, I think it's better to let `Device` worry about it.

In the meeting of Feb.17, https://openmmlab.feishu.cn/docs/doccnU1MVVe7JSBOj9pEHMi4CCf, we have decided not use vulkan device now, but only use a option to handle vulkan on/off in ncnn. I will consider reorganize the code in a better way.